java/src/main/java/org/rocksdb/MutableColumnFamilyOptionsInterface.java

```diff
@@ -26,7 +26,7 @@
    * @throws java.lang.IllegalArgumentException thrown on 32-Bit platforms
    *   while overflowing the underlying platform specific value.
    */
-  MutableColumnFamilyOptionsInterface setWriteBufferSize(long writeBufferSize);
+  MutableColumnFamilyOptionsInterface<T> setWriteBufferSize(long writeBufferSize);
```

Comment:
Why not just `T` as per the other functions?

Reply:
@adamretter That sounds sensible. It just didn't occur to me.